peribolos/knative-sandbox.yaml

```diff
@@ -505,6 +505,11 @@ orgs:
             - navidshaikh
             - rhuss
             privacy: closed
+      Contour Approvers:
```

Comment:
I'd call this `net-contour` and grant repo write access (helps with issue triage, and may be necessary for `CODEOWNERS`)

Reply:
forgot about issue triage